internal/server/ip/link_veth.go

```diff
@@ -1,29 +1,38 @@
 package ip
 
 import (
-	"github.com/lxc/incus/v6/shared/subprocess"
+	"github.com/vishvananda/netlink"
 )
 
 // Veth represents arguments for link of type veth.
 type Veth struct {
 	Link
 	Peer   Link
-	Master string
+	Master string // TODO: Link already has a master, and the way this one is used does not suggest that it is different
 }
 
 // Add adds new virtual link.
 func (veth *Veth) Add() error {
-	err := veth.Link.add("veth", append([]string{"peer"}, veth.Peer.args()...))
+	veth.Link.Master = veth.Master // TODO: this is just to see if it works
```

Comment:
Did that work? If so, then we could remove the Master from Veth and clear both TODO?